.pre-commit-config.yaml

```diff
@@ -1,8 +1,92 @@
 repos:
-  - repo: https://github.com/psf/black
-    rev: 22.12.0
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v5.0.0
     hooks:
-      - id: black
-        language_version: python3.12
-
-
+      - id: check-added-large-files
```

Comment:
Exactly. In case it is really necessary we can simply add an exclude to that specific hook ,i.e., 

`exclude: path/to/too_large_file.exe`

If the file should be much larger than 1MB  we should anyways think about using gitLFS.

Reply:
Perfect, I think that is a very good approach.